backend/usecases/chat/tool_executor.py

```
@@ -182,6 +185,140 @@ async def execute_loop(
+                        temperature=temperature,
+                        max_tokens=max_tokens,
+                    ):
+                        if chunk.type == "delta":
+                            # テキストデルタを yield
+                            yield chunk
+                        elif chunk.type == "tool_call" and chunk.tool_calls:
+                            # ツール呼び出しを蓄積
+                            has_tool_calls = True
+                            final_tool_calls.extend(chunk.tool_calls)
+                        elif chunk.type == "done":
+                            # 完了通知を yield しない（ループ制御）
```

Ruff (0.14.13)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)